pkg/common/utils.go

```diff
@@ -217,3 +224,26 @@ func ConvertLabelsStringToMap(labels string) (map[string]string, error) {
 
 	return labelsMap, nil
 }
+
+// Transform storage locations from a comma separated string to an array,
+// and validate the format of each location. Note that empty string does
+// not need specail treatment because Split() turns it into an empty
+// array, which is the desired value.
+func ProcessStorageLocations(storageLocations string) ([]string, error) {
+	locations := strings.Split(storageLocations, ",")
+
+	var normalizedLocations []string
+
+	multiRegionalPattern, _ := regexp.Compile(multiRegionalLocationFmt)
```

Comment:
I don't know how often this is called and if the patterns are known in advance and never changed probably these two calls to `regexp.Compile` could be const and declared at the top

Reply:
Ack. I'll pre-compute the pattern.

Reply:
Done.